docs/source/parallelism.mdx

```diff
@@ -209,6 +210,7 @@ Here the bubble (idle time) is further minimized by prioritizing backward passes
 
 Varuna further tries to improve the schedule by using simulations to discover the most efficient scheduling.
 
+OSLO has pipeline parallelism implementation based on the Transformers.
```

Comment:
If you think writing this way could make some problems, it's a good idea to include something you haven't tested yet. (But Internally, we succeeded in training the 1.3B GPT model using OSLO)